.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: cpp
+
+addons:
+  apt:
+    packages:
+    - swig
+    - python-dev
+    - libguile-dev
+    - octave-headers
+    - cmake
+
+script:
+  - ./autogen.sh --no-configure
+  - ./configure --prefix=$HOME/.local --enable-shared --enable-maintainer-mode --enable-cxx
+  - make -j2
+  - make install
+  - python ./test/test_std.py
```

Comment:
it's the default user python search path, used when you do "python setup.py install --user"